packages/fxa-payments-server/src/styles/tailwind.css

```diff
@@ -37,3 +37,60 @@
 .payment-error .legal-heading {
   @apply mt-5;
 }
+
+/* shared styles used in PlanDetails and PlanUpgradeDetails */
+.btn {
+  @apply border-transparent text-blue-500 leading-5 my-2 outline-none py-2 pr-2 pl-6 relative focus:border focus:border-solid focus:border-blue-400 focus:py-2 focus:pr-4 focus:pl-10 focus:shadow-none;
+}
```

Comment:
Please add `cursor-pointer` to improve a11y as there is no other type of state change to inform the user that it is a button.

Nit: I know you didn't come up with the class name, but if it's not a heavy lift (that is, I didn't check if other components or routes dependent on it aside from `PlanUpgradeDetails`, so please ignore if there are) can we possibly rename to "accordion-button" or "accordion-btn" or something a bit more specific?

Reply:
I'll add `cursor-pointer`, sorry I forgot to add it before!

`.btn` doesn't look like it's in anything else in `fxa-payments-server`, so it'll be an easy change to `accordion-btn`. I couldn't figure out how to convert the rest of it though. I'll take another look at it.

```
  .btn {
    background: none;
    height: initial;
    width: initial;
  }
```